python/src/opendp/extras/__init__.py

```diff
@@ -0,0 +1,3 @@
+'''
+The ``extras`` module is a namespace for functionality that requires extra, optional Python dependencies.
```

Comment:
Looking at #1782, I'm reminded that what defined `_extrinsics` was not the dependence on an extra install, it was that it was pure-python code: It just happened that the examples we had depended on outside libraries.

I'd suggest that the module description just be updated:
```suggestion
The ``extras`` module contains pure-python DP tools.
These are not yet supported by the Rust or R bindings.
 Some tools require extra, optional Python dependencies, and are grouped under corresponding namespaces.
```